packages/main/src/Select.ts

```diff
@@ -214,6 +214,18 @@ class Select extends UI5Element implements IFormInputElement {
 	@property({ type: Boolean })
 	disabled = false;
 
+	/**
```

Comment:
I think we have to add a note either here as part of the icon property, or some section as part of the component overview that when you set the `icon` property, the Select will display the given icon and will not display the selected option's label, e.g iconOnly appearance (If I got how it works).